src/foraging_gui/Foraging.py

```diff
@@ -3549,12 +3572,15 @@ def _Start(self):
             workerGenerateAtrial.signals.finished.connect(self._thread_complete4)
             workerStartTrialLoop = Worker(self._StartTrialLoop,GeneratedTrials,worker1,workerPlot,workerGenerateAtrial)
             workerStartTrialLoop1 = Worker(self._StartTrialLoop1,GeneratedTrials)
+            worker_save = Worker(self._Save,BackupSave=1)
+            worker_save.signals.finished.connect(self._thread_complete5)
```

Comment:
should this be `_thread_complete6` for consistency?

Reply:
Updated naming for consistency.